src/harness/testcases/WINNF_FT_S_MCP_testcase.py

```diff
@@ -80,6 +80,10 @@ def checkFadConfiguration(fad_config):
           len(config['domainProxyConfigs']),
           'Mismatch in the number of domain proxies and the configuration for this iteration: %s.'
           % str(iteration_data))
+      self.assertLessEqual(
```

Comment:
I think we want these to be equal, right?

Reply:
I agree, I chose the easiest way to have this check, but I agree that it is better to have more strict check, please check the update.